### ISSUE TITLE
{Freeway-Bug}[Adaptive Card iOS - ActivityUpdate.json]: Luminosity ratio for placeholder text 'Add a comment' with respect to background is '1.741.1' which is less than 4.5:1

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8408]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.0" control and select it.
4. Navigate to 'Scenarios' control and activate it.
5. Navigate to 'ActivityUpdate.json' under 'Elements' list item and activate it.
6. Card will be appear.
7. Navigate to 'Set due date' button and activate it.
8. Navigate to 'Add a comment' text field
9. Observe the issue for 'Add a comment' text using color contrast analyzers.

**Actual Result:**
Luminosity ratio for placeholder text 'Add a comment' is **'1.741.1'** with respect to background, which is less than 4.5:1.

**Expected Result:**
Luminosity ratio for placeholder text 'Add a comment' with respect to background should be equal to or greater than 4.5:1

**Notes:**
1. v1.0-Scenarios-InputForm.json-Luminoisty ratio for placeholder text 'your email' is less (Please find attachment)
2. Issue is repro throughout the application with placeholder text like this.

**User Experience:**
When a minimum contrast ratio is not provided between text and its background, it is difficult for low-vision users as well as anyone in extreme lighting environments to read the content.  

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

<img width="960" alt="WCAG1 4 3 Luminosity ratio for Place holder text 'Add a comment' with respect to it's background is less" src="https://user-images.githubusercontent.com/95913748/227851235-e599c6b3-f7a6-4805-9e78-01684d6fc7df.png">
<img width="960" alt="Note-v1 0-Scenarios-InputForm json-Luminoisty ratio for placeholder text 'your emal' is less" src="https://user-images.githubusercontent.com/95913748/227851242-d3aeffa4-2a6c-4a2e-96e1-44fb7e395ae8.png">

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
Closing this issues for two reasons.
1. A11y Tests should not be run against Elements folders.
2. it turned out that card is moved to Scenario since the last test, and the issue is already fixed.
![Screenshot 2023-05-25 at 1 04 10 PM](https://github.com/microsoft/AdaptiveCards/assets/4112696/c6434e73-e478-4d15-9254-02a6cb087e1e)
![Screenshot 2023-05-25 at 1 04 24 PM](https://github.com/microsoft/AdaptiveCards/assets/4112696/d11f5e87-878a-4130-b05b-a762d39bb4f6)

**Comment by vagpt** [comment]
As per the above comment and the placeholder text is not available, hence closing this issue.

**Build Version: 1.0 (2.4.0-beta.23.6.4.1)**
